src/main/kotlin/dev/arbjerg/ukulele/command/QueueCommand.kt

```diff
@@ -0,0 +1,30 @@
+package dev.arbjerg.ukulele.command
+
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack
+import dev.arbjerg.ukulele.audio.PlayerRegistry
+import dev.arbjerg.ukulele.jda.Command
+import dev.arbjerg.ukulele.jda.CommandContext
+import org.springframework.stereotype.Component
+
+@Component
+class QueueCommand (
+        private val players: PlayerRegistry
+) : Command("q") {
+
+    override suspend fun CommandContext.invoke() {
+        reply(printQueue(players[guild].getQueue()))
+    }
+
+    private fun CommandContext.printQueue(queue: List<AudioTrack>): String {
+        val sb = StringBuilder()
+        when (queue.size) {
```

Comment:
Having a when block doesn't make sense here.
1. You can just do `if (queue.isEmpty() return "Empty queue"`
2. Even then, an if/else would look nicer